lib/active_remote/cached.rb

```diff
@@ -12,6 +12,8 @@ module ActiveRemote
   module Cached
     extend ::ActiveSupport::Concern
 
+    RUBY_AND_ACTIVE_SUPPORT_VERSION = "#{RUBY_VERSION}:#{RUBY_PLATFORM}:#{ActiveSupport::VERSION::STRING}".freeze
```

Comment:
@liveh2o just wanted to point out here the ruby platform and hear any thoughts?

Reply:
I am curious about the utility of adding the platform. But ultimately, I think the worst case scenario is that there is a cache miss when this rolls out. I'm cool moving forward.

Reply:
This might be better if I used `RUBY_ENGINE_VERSION` instead of `RUBY_VERSION` & `RUBY_PLATFORM`. 

Because in  jruby-9.2.6.0 `RUBY_VERSION = 2.5.3`, and `RUBY_ENGINE_VERSION=9.2.6.0`. But in MRI, RUBY_VERSION and RUBY_ENGINE_VERSION are identical.